tests: Add SPIRV-Hopper (#5913)
KhronosGroup/Vulkan-ValidationLayers: tests/spirv_hopper/external/README.md

# SPIRV-Reflect

> These files are taken from https://github.com/KhronosGroup/SPIRV-Reflect

Reviewer: Can we not do this please?

Bugfixes and potentially security fixes get stranded.

Author: Would it be better for the short term to hide this `SPIRV-Hopper` behind an extra Opt-In flag

Reviewer 2: The best solution is to upstream good CMake support IMO. I'd rather not add an extra Opt-In flag.

Reviewer 3: > The best solution is to upstream good CMake support IMO. I'd rather not add an extra Opt-In flag.

Should we do the same for VMA?

Reviewer 3: Also, this needs to be a new issue, not a discussion on a closed PR

The reasons to check these in instead of adding to `known_good.json` are

- This repo is not updated often

> - It is designed to be a files people include in their build

Reviewer 4: ```suggestion
- These files are designed for people to include in their build
```
